src/Form/Form.test.js

```diff
@@ -194,12 +191,48 @@ describe('Test the form class reading values correctly', () => {
     <label for="frmCCNum">Card Number</label>
     <input name="cardnumber" id="frmCCNum" autocomplete="cc-number" value="4111111111111111">
     <label for="frmCCExp">Expiry</label>
-    <input name="cc-exp" id="frmCCExp" placeholder="MM-YYYY" autocomplete="cc-exp" value="12-2028">
+    <input name="cc-exp" id="frmCCExp" placeholder="YYYY-MM" autocomplete="cc-exp" value="2028-12">
```

Comment:
Here I have inverted `MM-YYYY` to `YYYY-MM` so we also check that use case (also covered in specific unit tests).